test/cypress/integration/29-host.dashboard.js

```diff
@@ -8,4 +8,81 @@ describe('host dashboard', () => {
     cy.get('.Orders .item:first .status').contains('paid');
     cy.wait(1000);
   });
+
+  it('approve expense and reject expense', () => {
+    cy.login({ redirect: '/brusselstogetherasbl/dashboard/expenses' });
+    // navigate to first item
+    cy.get('[data-cy="expense-paid"]').as('currentExpense');
+    cy.get('[data-cy="expense-actions"]')
+      .as('currentExpenseActions')
+      // click 'unapprove'
+      .contains('button', 'Unapprove')
+      .click({ force: true });
+    // modal pops up - click 'yes'
+    cy.get('[data-cy="confirmation-modal-continue"]').click({ force: true });
+    cy.wait(5000);
```

Comment:
That's a long time to wait, is it necessary?

Reply:
Probably not - I think this failed once or twice because it moved on too quickly from the modal, so I put in a wait, but I can minimise the wait and see how that goes.